Makefile

```diff
@@ -0,0 +1,18 @@
+fmtcheck:
```

Comment:
I can't help but notice that this file is itself missing a copyright header 😆

Reply:
Good catch! I don't think it matters _at this point_.

The reason is that the tool basically does not know how to add the header to makefiles, it's not that it would be intentionally _ignored_.

See https://github.com/hashicorp/copywrite/blob/0f077594eee1ea09d721a7e6fc9f07302aaff7fe/addlicense/main.go#L360-L392

Also the same issue applies to the upstream project our `copywrite` has forked, although it does have some pending PRs:
 - https://github.com/google/addlicense/pull/147
 - https://github.com/google/addlicense/pull/104
 - https://github.com/google/addlicense/pull/54

To be fair there is support for `*.cmake` but I don't have enough context/knowledge to tell what that is and what's the relationship with `Makefile` or `GNUMakefile` and I am not quite convinced there's a value in learning that just to add the license headers. I would perceive the licensing risk with makefiles in general to be relatively low in comparison with the rest of the codebase.

Feel free to correct me if I'm wrong in any of the above though!

Reply:
Agreed, it seems pretty harmless to leave the Makefile un-headered!